### PR TITLE
Feature/ubi 243 gke deployment

### PR DESCRIPTION
**Pull request by MaximeStOnge**
#### What changes does this PR contain?
- Resources override
- Readme
- Elasticsearch connection
- Enable HotRod tool for testing

#### Which JIRA ticket ID's are associated with this PR?
UBI-243

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  elasticsearch:
+    scheme: http
+    host: elasticsearch-master
+    port: 9200
+    user: elastic
+    usePassword: false
+
+agent:   
+  resources:
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:16]: Always put the same order for request->limits... req first.

File: chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      cpu: 500m
+      memory: 512Mi
+    requests:
+      cpu: 25m #default 256m
+      memory: 128Mi
+
+collector:
+  resources:
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:25]: Same here

File: chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      cpu: 1
+      memory: 1Gi
+    requests:
+      cpu: 50m #default 500m
+      memory: 512Mi
+
+query:
+  resources:
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:34]: Same here

File: chart/values.yaml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      cpu: 500m
+      memory: 512Mi
+    requests: 
+       cpu: 10m #default 256m
+       memory: 128Mi
+
+esIndexCleaner:
+  enabled: true
+  resources:
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:44]: Same here

File: chart/values.yaml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+esRollover:
+  enabled: true
+  extraEnv:
+    - name: CONDITIONS
+      value: '{
+        "max_age": "30d",
+        "max_size": "30GB" 
+      }'
+  resources:
+    limits:
```
Comment (ysma500) [inline chart/values.yaml:60]: Same here

File: chart/values.yaml [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    requests:
+      cpu: 10m #default 256m
+      memory: 128Mi
+
+esRollover:
+  enabled: true
+  extraEnv:
+    - name: CONDITIONS
+      value: '{
+        "max_age": "30d",
```
Comment (ysma500) [inline chart/values.yaml:56]: We can put a smaller rollOver like 7days and 2Gi for now

File: chart/values.yaml [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+      cpu: 10m
+      memory: 128Mi
+
+# Total
+#   limits:
+#     cpu: 220m
+#     memory: 626Mi
+#   requests:
+#     cpu: 800m
+#     memory: 596Mi
```
Comment (ysma500) [inline chart/values.yaml:94]: New line EOF missing

**Review by ysma500 — CHANGES_REQUESTED** [review]

File: chart/values.yaml (hunk 1 above)
Comment (MaximeStOnge) [reply chart/values.yaml:16]: Logique, j'ai copié des values.yaml de chaque outil et ils utilisent pas la même convention j'ai pas porté attention

**Review by ysma500 — APPROVED** [review]

**Review by ArcticWalrus — APPROVED** [review]

**Pull request merged**